src/NITTA/Model/Networks/Bus.hs

```diff
@@ -86,6 +114,7 @@ busNetwork name iosync =
         , bnSignalBusWidth = 0
         , ioSync = iosync
         , bnEnv = def
+        , bnPUPrototypes = M.empty
```

Comment:
Don't sure what is better, but you can use `def` here.

Reply:
Used `def`  to reduce code changes in future